.github/workflows/web-test-code-quality.yml

```diff
@@ -0,0 +1,73 @@
+name: Web code quality checks
+on:
+  pull_request:
+    paths:
+      - ".github/workflows/web-test.yml"
+      - "web-admin/**"
+      - "web-auth/**"
+      - "web-common/**"
+      - "web-local/**"
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Filter modified codepaths
+        uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            admin:
+              - ".github/workflows/web-test.yml"
+              - "admin/**"
+              - "cli/**"
+              - "runtime/**"
```

Comment:
We don't need the `web` code quality tests to run when `cli` or `runtime` code changes